CODE_OF_CONDUCT.md

```diff
@@ -1,4 +1,3 @@
-# Kuma Community Code of Conduct
-
-Kuma follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+# Kuma community code of conduct
 
+Kuma follows the [CNCF Community Code of Conduct v1.2](https://github.com/cncf/foundation/blob/main/code-of-conduct.md).
```

Comment:
Don't change this it's synced from github.com/kumahq/.github

Reply:
Oh, I didn’t know that. Reverting this then.